Mathlib/Algebra/Lie/Killing.lean

```diff
@@ -411,6 +441,11 @@ lemma restrictBilinear_killingForm (H : LieSubalgebra R L) :
     H.restrictBilinear (killingForm R L) = LieModule.traceForm R H L :=
   rfl
 
+@[simp] lemma ker_traceForm_of_isCartanSubalgebra_eq_bot
```

Comment:
Shouldn't the name be
```suggestion
@[simp] lemma ker_traceForm_eq_bot_of_isCartanSubalgebra
```
I guess the same applies to `ker_restrictBilinear_of_isCartanSubalgebra_eq_bot` which must have slipped through during the review of an earlier PR.

Reply:
Good point, done.